app/components/candidate_interface/restructured_work_history_work_experience_component.html.erb

```diff
@@ -22,7 +22,7 @@
       <%= link_to  candidate_interface_edit_restructured_work_history_path(@work_experience.id), class: 'govuk-link' do %>
         Change <span class="govuk-visually-hidden">entry for experience between <%= formatted_start_date %> and <%= formatted_end_date %></span>
       <% end %>
-
+        &nbsp;&nbsp;&nbsp;
```

Comment:
There might be a better way that @paulrobertlloyd  or @frankieroberto can think of, but we can use govuk-!-margin-right-3 here instead.

Reply:
Ooompf @frankieroberto, what’s going on here!? (This is in the prototype). A short term fix would be to use `govuk-!-margin-right-2`.

Worth noting that in the summary card component, multiple actions are rendered in an unordered list. Looking at the CSS for that component: https://github.com/alphagov/govuk-frontend/blob/803da2c92d1a73401ac506b2dc8a393fae940508/src/govuk/components/summary-list/_index.scss#L86-L108. Maybe we should use similar markup here?

Reply:
@paulrobertlloyd opps, this was just a quick hack in the prototype to separate out the actions links a bit more, (old school! 😆). Agree that `govuk-!-margin-right-2` on the first link would be fine for now.  Could use a UL – I'm never sure whether the extra structure is useful for screen readers in these scenarios or just adds verbosity? 🤷‍♂️